go-controller/go.mod

```diff
@@ -48,6 +50,7 @@ require (
 
 replace (
 	github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
+	github.com/ovn-org/libovsdb => github.com/ovn-org/libovsdb v0.5.1-0.20210624111205-868f9a0c2b94
```

Comment:
you might as well just bump the the main dep to this commit - we can ensure we're using a tagged version if libovsdb before we make a release.